### DISCUSSION TITLE
Options for datacenter

### DISCUSSION THREAD

**Discussion by albiejvr** [community/community#39359] [Enterprise]
I'm doing some comparisons between GitHub and Azure DevOps. I am considering subscribing to the Team option for my company. 

One of the items on my checklist is localized hosting. I'm based in South Africa, and Azure has 2 datacenters in the country (SA North and South). Does GitHub have a similar option? 

I haven't managed to find a good map of GitHub server locations.

**Comment by sr229** [comment]
GitHub has the same coverage as Azure regions as it is a managed service. If you're looking to explicitly host it on the closest Azure region, then GitHub Enterprise Server may be for you.
